Translate plugin/progress-plugin.mdx

jiwoo0629 opened the pull request:
## Summary 

#7 

## Glossary

report progress : 진행 상황을 보고하다
progress message : 진행 메세지
progress percentage : 진행률
warm the cache : 데이터 값을 메모리에 캐시하다

jiwoo0629 committed:
Translate plugin/process-plugin.mdx

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/line/webpack.kr?pullRequest=328) <br/>All committers have signed the CLA.

jiwoo0629 committed:
Translate plugin/progress-plugin.mdx

commit한 글의 이름이 process로 잘못되어 있어 수정합니다.

choilim commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -67,19 +69,19 @@ new webpack.ProgressPlugin({
+이러한 문제를 해결하기 위해 `ProgressPlugin`은 마지막으로 알려진 총 modules 수를 캐시하여 이 값을 다음 빌드 때 재사용합니다. 첫번째 빌드는 데이터 값을 메모리에 캐시하지만 그 다음 빌드들은 이 값을 사용하고 업데이트합니다.
 
-> We recommend using `percentBy: 'entries'` setting for projects with [multiple configured entry points](/configuration/entry-context/#entry). Percentage calculation will become more accurate because the amount of entry points is known in advance.
+> [multiple configured entry points](/configuration/entry-context/#entry)가 있는 프로젝트들을 위한 설정에는 `percentBy: 'entries'`를 사용할 것을 추천합니다. 진입점의 양이 사전에 알려져 있기 때문에 확률 계산은 더욱 정확해질 것입니다.
```
`entry points` 는 다른 번역 파일에선 `엔트리 포인트`로 통합하고 있습니다~

jiwoo0629 replied:
수정했습니다~

jiwoo0629 committed:
entry points -> 엔트리 포인트로 수정

entry points -> 엔트리 포인트로 수정했습니다.

dkstyle commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -37,17 +39,17 @@ new webpack.ProgressPlugin(handler);
+- `activeModules` (`boolean = false`): 활성화된 모듈의 수와 활성화된 모듈 하나를 진행 메세지로 보여줍니다.
+- `entries` (`boolean = true`): entries 수를 진행 메세지로 보여줍니다.
+- `handler` ([Providing function](#providing-function)을 보세요)
+- `modules` (`boolean = true`): modules 수를 진행 메세지로 보여줍니다.
```
modules는 44번 라인과 같이 `모듈`로 번역하는 것이 일관성이 있어보입니다. (하위의 다른 컨텐츠도 동일)

jiwoo0629 committed:
modules -> 모듈 변경

line 47의 modules를 모듈로 변경했습니다

dkstyle commented:
https://github.com/line/webpack.kr/pull/328/files#diff-b43e6d83b72d9e00985a812021ea308f4c65a9f0712a09f87b84bcef395ad8e3R33

// e.g. Output each progress message directly to the console:
코드예제의 경우 주석내용은 번역해 주세요.

dkstyle commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -37,17 +39,17 @@ new webpack.ProgressPlugin(handler);
+- `entries` (`boolean = true`): entries 수를 진행 메세지로 보여줍니다.
+- `handler` ([Providing function](#providing-function)을 보세요)
+- `modules` (`boolean = true`): 모듈 수를 진행 메세지로 보여줍니다.
+- `modulesCount` (`number = 5000`): 시작할 modules 수의 최솟값이며 `modules` 속성이 활성화되면 적용됩니다.
```
다른 부분도 modules 번역을 통일 해주세요.

jiwoo0629 replied:
넵 다시 수정했습니다

jiwoo0629 committed:
modules -> 모듈 변경

함수 이름을 제외한 modules -> 모듈 변경

dkstyle commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -37,17 +39,17 @@ new webpack.ProgressPlugin(handler);
+- `modules` (`boolean = true`): 모듈 수를 진행 메세지로 보여줍니다.
+- `modulesCount` (`number = 5000`): 시작할 modules 수의 최솟값이며 `modules` 속성이 활성화되면 적용됩니다.
+- `profile` (`boolean = false`): 진행 단계의 프로필 정보를 수집하도록 `ProgressPlugin`에게 알려줍니다.
+- `dependencies` (`boolean = true`): dependencies 수를 진행 메세지로 보여줍니다.
```
dependencies의 경우 다른 `의존성`/`종속성`/`디펜던시` 표현을 컨텐츠에 따라 의미에 맞게 선택해서 번역하면 좋을 것 같습니다.

jiwoo0629 replied:
번역 수정 및 코드 예제 내 코드 설명 번역 완료했습니다~

jiwoo0629 committed:
내용 수정 및 번역

line 33 : entries -> 엔트리 변경
dependencies -> 종속성 변경
코드 예제 내 주석 변역 완료

dkstyle reviewed (approved):
LGTM 👍

dkstyle commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -37,24 +39,24 @@ new webpack.ProgressPlugin(handler);
   entries: true,
   handler(percentage, message, ...args) {
-    // custom logic
+    // 사용자 지정 논리
```
사족이지만 logic은 `논리` 보다 `로직`으로 그대로 표현하는것이 더 자연스러운 것 같습니다.

jiwoo0629 replied:
다시 생각해보니 저도 그렇게 느껴서 변경했습니다 감사합니다!

jiwoo0629 committed:
논리 -> 로직 변경

논리 -> 로직 변경

YukJiSoo commented on src/content/plugins/progress-plugin.mdx:
```diff
@@ -67,19 +69,19 @@ new webpack.ProgressPlugin({
+이러한 문제를 해결하기 위해 `ProgressPlugin`은 마지막으로 알려진 총 모듈 수를 캐시하여 이 값을 다음 빌드 때 재사용합니다. 첫번째 빌드는 데이터 값을 메모리에 캐시하지만 그 다음 빌드들은 이 값을 사용하고 업데이트합니다.
 
-> We recommend using `percentBy: 'entries'` setting for projects with [multiple configured entry points](/configuration/entry-context/#entry). Percentage calculation will become more accurate because the amount of entry points is known in advance.
+> [multiple configured 엔트리 포인트](/configuration/entry-context/#entry)가 있는 프로젝트들을 위한 설정에는 `percentBy: 'entries'`를 사용할 것을 추천합니다. 진입점의 양이 사전에 알려져 있기 때문에 확률 계산은 더욱 정확해질 것입니다.
```
`entry points` 를 `엔트리 포인트`로 번역하여 사용하고 있기 때문에 `진입점`으로 번역된 부분 수정이 필요해보입니다!

그리고 `진입점의 양이 사전에 알려져 있기 때문에` 보다 `엔트리 포인트의 개수를 미리 알 수 있기 때문에` 로 번역하는게 더 자연스럽게 읽혀질 것 같은데 어떠신가요?

jiwoo0629 replied:
수정 완료했습니다!

jiwoo0629 committed:
진입점 -> 엔트리 포인트 수정

진입점-> 엔트리 포인트로 수정하고 이에 따라 line 78의 번역을 수정했습니다.
또 해당 줄의 multiple configured -> 다중 구성된 번역을 진행했습니다.
